jbpm-audit/src/test/java/org/jbpm/process/audit/jms/AsyncAuditLogProducerTest.java

```diff
@@ -484,7 +482,7 @@ public void onMessage(Message message) {
             };
             consumer.setMessageListener(rec);
             // since we use message listener allow it to complete the async processing
-            Thread.sleep(2000);
+            Thread.sleep(waitTime);
```

Comment:
Although I like parameterization of Thread.sleep, I can see the waiting raised from 2 secs to 5. Is this the only way to make the tests stable? It prolongs their execution.